testutil/src/main/java/org/spine3/test/Verify.java

```diff
@@ -184,25 +184,37 @@ public static void fail(String message, Throwable cause) {
     }
 
     /**
-     * Asserts that two floats are not equal concerning a delta. If the expected value is infinity then the delta value
-     * is ignored.
+     * Asserts that two floats are not equal concerning a delta. If the expected value is infinity or NaN then
```

Comment:
Use `{@code}` to wrap `infinity` and `NaN` in the JavaDoc sections of this class.